src/services/search.ts

```diff
@@ -36,9 +36,9 @@ export function fuzzySearch(haystack: string[], query: string, dispatcher: (data
       haystackOrder.push(haystack[info.idx[infoIdx]]);
     }
 
-    dispatcher([haystackOrder, [...matchesSet]]);
+    callback([haystackOrder, [...matchesSet]]);
   } else if (!query) {
-    dispatcher([[], []]);
+    callback([]);
   }
 }
 
```

Comment:
I was expecting a higher debouce value,  but it's 300. My only comment was that it felt a tiny, little, pinch, slow to respond to the filter. Like it's doing a very fast XHR request.


https://github.com/grafana/explore-logs/assets/1069378/6fc59968-5643-4156-8dde-a907154ac48f

Reply:
Agreed, we could decrease the debounce duration, but then the UI updates while you're still typing, or increase the debounce duration but then it makes it feel slow to update.

I could be convinced to change it, but let's merge it for now and follow up later?

Reply:
🤝